public/app/features/dashboard/components/GenAI/GenAIDashDescriptionButton.tsx

```diff
@@ -24,10 +24,14 @@ const DESCRIPTION_GENERATION_STANDARD_PROMPT =
 
 export const GenAIDashDescriptionButton = ({ onGenerate, dashboard }: GenAIDashDescriptionButtonProps) => {
   const messages = React.useMemo(() => getMessages(dashboard), [dashboard]);
-  const onClick = React.useCallback(() => reportGenerateAIButtonClicked(EventSource.dashboardDescription), []);
 
   return (
-    <GenAIButton messages={messages} onGenerate={onGenerate} onClick={onClick} loadingText={'Generating description'} />
+    <GenAIButton
+      messages={messages}
+      onGenerate={onGenerate}
+      loadingText={'Generating description'}
+      eventTrackingSrc={EventTrackingSrc.dashboardDescription}
```

Comment:
Should we also include the toggleTipTitle prop for generate dash title/ description / changes as well?

right now they just have a blank area where title would be
<img width="1113" alt="Screenshot 2023-10-05 at 12 08 27 AM" src="https://github.com/grafana/grafana/assets/22381771/8be8346f-acae-484b-8416-58bdce2a228f">